src/acl.c

```diff
@@ -2495,6 +2510,9 @@ void ACLFreeLogEntry(void *leptr) {
  * If `object` is not NULL, this functions takes over it.
  */
 void addACLLogEntry(client *c, int reason, int context, int argpos, sds username, sds object) {
+    /* Update ACL info metrics */
+    ACLUpdateInfoMetrics(reason);
```

Comment:
i wanna note that this will also affect RM_ACLAddLogEntry, but i suppose that's desired.
i.e. normally if the module does some ACL checks and decides to skip things without any logging, then these stats will not increment, i guess that's what we want.